ports/stm32/boards/ARDUINO_NICLA_VISION/pins.csv

```diff
@@ -166,6 +166,20 @@ PK4,PK4
 PK5,PK5
 PK6,PK6
 PK7,PK7
+
+# Arduino digital pins
+D0,PG12
+D1,PA9
+D2,PA10
+D3,PG1
+D4,PC7
+D5,PC6
```

Comment:
```suggestion
```
Note those two pins are broken out on the camera connector. I don't think it makes much sense to define them, since the camera connector isn't a generic B2B connector, but if we do then we should define the other pins on that connector.

Reply:
These pins have not been added.